.gitignore

```diff
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
+.vscode
```

Comment:
```suggestion
/.vscode
```

Reply:
I left out the forward slash because I want to ignore .vscode directories at every level, not only the top level. Does that make sense? Or should I still add the /?

Reply:
Oh, sorry. I thought that `.vscode` is only placed at the top directory.
If we may have `.vscode` in subdirectories, this configuration (no `/`) is preferred.